dyno-demo/src/main/resources/demo.properties

```diff
@@ -4,6 +4,8 @@
 LOCAL_DATACENTER=us-east-1
```

Comment:
Do we want to check this in as well? Or was this meant only for your local env?

Reply:
Oh! this was meant to be testing only. This file should not have any changes; I will revert them.